pom.xml

```diff
@@ -0,0 +1,138 @@
+<project xmlns="http://maven.apache.org/POM/4.0.0" xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance" xsi:schemaLocation="http://maven.apache.org/POM/4.0.0 http://maven.apache.org/maven-v4_0_0.xsd">
+
+  <modelVersion>4.0.0</modelVersion>
+
+  <name>${project.groupId}:${project.artifactId}</name>
+  <description>reportportal Collector currently collects data from report portal</description>
+  <url>https://github.com/capitalone/Hygieia</url>
+  <packaging>jar</packaging>
+
+  <artifactId>reportportal_test_results_collector</artifactId>
+
+  <parent>
+    <groupId>com.capitalone.dashboard</groupId>
+    <artifactId>Hygieia</artifactId>
+    <version>3.0.2-SNAPSHOT</version>
+    <relativePath>../../..</relativePath>
```

Comment:
please remove this dependency on relativePath. Every collector should be on its own.